src/Patterns/PatternImages.php

```diff
@@ -29,6 +29,77 @@ public function get_patterns_dictionary() {
 		return wp_json_file_decode( $patterns_dictionary, array( 'associative' => true ) );
 	}
 
+	/**
+	 * Returns the pattern images.
+	 *
+	 * @param string $pattern_slug The pattern slug.
+	 *
+	 * @return array The pattern images.
+	 */
+	public static function get_pattern_images( string $pattern_slug ): array {
```

Comment:
It feels to me like there is plenty of logic in here that would benefit from a unit test. What do you think?

Reply:
Totally agree, there was a `PatternImagesTest.php` with some tests for the files, but it was temporarily deleted (for more context see https://github.com/woocommerce/woocommerce-blocks/pull/10881#issuecomment-1721496102).
I'll open an issue to re-enable them and add unit tests for those two `get_image_url` and `get_pattern_images` functions, do you think that's okay?

Reply:
Here's the new issue, I added it to the epic https://github.com/woocommerce/woocommerce-blocks/issues/10987

Reply:
Yep, no probs :)